webapp/src/components/data/dataset_input/DatasetInput.vue

```diff
@@ -192,7 +200,7 @@ import FileSelection from './FileSelection.vue'
 interface Props {
   task: Task
   datasetBuilder: data.DatasetBuilder<File>
-  isOnlyPrediction: boolean //TODO: seems to always be false
+  isOnlyPrediction: boolean //is true when predicting with a trained model
```

Comment:
haha, yeah, I marked as TODO a while back and it got out-of-sync with my current understanding, thanks!

Reply:
Oh I thought I was the one to write the previous TODO haha